### ISSUE TITLE
Option for "Touch screen sides" to instantly go to next/previous image

### ISSUE THREAD

**Issue by marcosilva85** [deckerst/aves#289]
**Is your feature request related to a problem? Please describe.**
If possible, could you add an option for people like me to not have to swipe left/right for previous/next photo?
I have some albums with hundreds of pictures and when in full-screen swiping does not allow to instantly preview differences in similar photos and is too tedious if you intend to start photo curating/organizing session with Aves.

**Describe the solution you'd like**
An option to enable (if we want) to use a region on each side in the screen to instant transition to the previous/next photo - with no animation.
Regarding the buttons toolbar, it would be nice to keep showing/hiding the toolbar even during the instant transitions. We should still be able to use the middle of the screen to change its state to hidden or showing (as one prefers it).

**Describe alternatives you've considered**
If you want to test this in another app that has this in action, in SimpleGallery by (Tibor Kaputa), in its settings the option is called:
"Allow instantly changing media by clicking on screen sides."
It works for both Images and videos.

**Additional context**
For someone that's into photography is a life-changing addition, because with it we can quickly compare back and forth 5-10 pictures taken in burst mode, for example to check slight differences / delete some / leave the best.

Thank you for Aves, it is wonderful already.

**Comment by deckerst** [comment]
It's something I'd like to do (it's already part of #180 ).

In the meantime, you could show the thumbnails on the viewer overlay. Tapping on the thumbnails shows the items instantly.

![Screenshot_20220722-124810_Aves](https://user-images.githubusercontent.com/13898333/180423930-0c92bf3f-24bb-4e9a-950d-514af50ddcab.png)

**Comment by marcosilva85** [comment]
Interesting. Never saw that thumbnail feature before, thanks for the suggestion. It reminds me immediately of the iOS gallery app which also allows instant change. I've tried it and I hate to admit 🫣😅 but I noticed that the iOS feels more "responsive", i.e., you slide the "carrousel" and it shows very very quickly any of the images that are being scrolled through, its amazingly snappy to our fingers movement.

Any thoughts if that is possible to achieve in Android, or, if your intention is going on the same direction?

Thanks.

**Comment by deckerst** [comment]
The Samsung Gallery on Android provides overlay thumbnails with instant preview when swiping them, just as you describe.

I'd like to do the same, but currently the way images are loaded does not gel well with high speed change, so I introduced the delay you noticed. I figured that if you swipe through the thumbnails you're not really seeing the images anyway, unless you're making a kineograph! That said, I intend to come back to this eventually and make it snappy if I can.
